core/src/main/java/com/amplifyframework/core/model/query/predicate/BeginsWithQueryOperator.java

```diff
@@ -20,14 +20,14 @@
 /**
  * Represents a begins with condition with a target value for comparison.
  */
-public final class BeginsWithQueryOperator extends QueryOperator {
-    private Object value;
+public final class BeginsWithQueryOperator extends QueryOperator<String> {
```

Comment:
If the constructor can be package-only access, can the class, too?

Reply:
Yep it definitely can be and should be.